qa/suites/upgrade/pacific-x/stress-split/2-first-half-tasks/rbd-cls.yaml

```diff
@@ -3,7 +3,7 @@ meta:
    run basic cls tests for rbd
 first-half-tasks:
 - workunit:
-    branch: octopus
+    branch: quincy
```

Comment:
I'm not sure I understand why this is set to `quincy` instead of `pacific`?  The test binaries would come from `pacific` due to
```
- install:
    branch: pacific
    exclude_packages:
      - ceph-volume
```
in `qa/suites/upgrade/pacific-x/stress-split/1-start.yaml`, wouldn't they?  Or am I looking at the wrong place?

Reply:
Mapping the logic that you seem to have followed in this PR to https://github.com/ceph/ceph/pull/50759, all such stanzas that would have to point to `main` there.  And yet:
- here it's set to `quincy` throughout (regardless of whether it's an `octopus-x` or `pacific-x` suite)
- there it's set to either `pacific` or `quincy`, depending on whether it's a `pacific-x` or `quincy-x` suite

Can you explain this difference in approaches?

Reply:
@idryomov 
we had used workunits branch incorrectly and tests work because units don't change often.

However, the logic has to be as: 

for tests like `<branch>-x` in all `rbd-*` tests should use `<branch>` for workunits

@jdurgin @neha-ojha pls confirm this logic sounds correct.

(I will fix other PRs after we agree on this one)